### PR TITLE
fix(docs): Improved GCP tutorial

### PR DESCRIPTION
**Pull request by intls**
Fixes issue #2173 and some additional improvements
My Warpcast username: @pashu777

## Why is this change needed?

Users often encounter errors during the installation process due to missing information in the documentation. This PR fixes several issues in the Farcaster Hub installation guide for Google Cloud Platform (GCP). The changes include adding missing steps, clarifying instructions, and fixing grammatical errors to ensure a smooth setup experience for users.

### Change Summary

1. Enable Compute Engine API
   - Problem: After running the terraform apply command, users encounter an error indicating that the Compute Engine API is disabled. This PR includes a step to enable the API to prevent this error.
  
GCP Error

   - Solution: Added a step to enable the Compute Engine API before proceeding with the installation. Included an illustrative screenshot for guidance.
gcp_compute_engine_api

2. GCP Configuration Clarity
   - Problem: When setting up the GCP virtual machine for the Farcaster project, the configuration file ([main.tf](https://main.tf/)) includes the placeholder $YOUR_PROJECT_ID. Users might mistakenly copy and paste the configuration text without replacing $YOUR_PROJECT_ID with their actual project ID, leading to errors.
   - Solution: Added text before the GCP virtual machine configuration copy step to instruct users to replace $YOUR_PROJECT_ID with their personal ID       
gcp_project_id

3. Grammar Correction
   - Problem: Redundant text in the instructions: "Now follow the steps as described on Install page page."
   - Solution: Removed the redundant word "page" for improved readability.
page page

## Merge Checklist

_Choose all relevant options below by adding an `x` now or at any time before submitting for review_

- [x] PR title adheres to the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) standard
- [ ] PR has a [changeset](https://github.com/farcasterxyz/hub-monorepo/blob/main/CONTRIBUTING.md#35-adding-changesets)
- [x] PR has been tagged with a change label(s) (i.e. documentation, feature, bugfix, or chore)
- [x] PR includes [documentation](https://github.com/farcasterxyz/hub-monorepo/blob/main/CONTRIBUTING.md#32-writing-docs) if necessary.

---

## PR-Codex overview
The focus of this PR is to update the main.tf file with instructions to configure a GCP virtual machine and enable the Compute Engine API.

### Detailed summary
- Added configuration for GCP virtual machine with image references
- Instructed to replace "$YOUR_PROJECT_ID" with personal ID
- Provided steps to enable Compute Engine API
- Updated instructions for installing Docker
- Clarified steps to follow for setting up Hubble

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: apps/hubble/www/docs/tutorials/gcp.md [hunk 1]
```diff
@@ -26,7 +26,11 @@ Execute the below commands in the cloud shell:
 nano main.tf
 </code></pre>
 
-Now paste the content of below into your main.tf \
+Now paste the content of below into your main.tf \ 
+Replace "$YOUR_PROJECT_ID" with your personal ID.
+
+<figure><img src="../assets/images/gcp_project_id.png"
```
Comment (sds) [inline apps/hubble/www/docs/tutorials/gcp.md:32]: This is missing ` /></figure>` to close the `figure` and `img` tags. This breaks the build—please fix.

File: apps/hubble/www/docs/tutorials/gcp.md [hunk 2]
```diff
@@ -92,6 +96,10 @@ Example output:
 
 <figure><img src="../assets/images/gcp_terraform_plan.png" alt=""><figcaption><p>Example output of terraform plan</p></figcaption></figure>
 
+Enable Compute Engine API
+
+<figure><img src="../assets/images/gcp_compute_engine_api.png"
```
Comment (sds) [inline apps/hubble/www/docs/tutorials/gcp.md:101]: Same issue here.

**Review by sds — COMMENTED** [review]
Looks like build is broken. Can merge once fixed—thank you.

File: apps/hubble/www/docs/tutorials/gcp.md (hunk 1 above)
Comment (intls) [reply apps/hubble/www/docs/tutorials/gcp.md]: @sds, I have fixed the errors in the figure and img tags as you suggested. The pull request is now ready for review and merging. Please review the changes. Thank you!

File: apps/hubble/www/docs/tutorials/gcp.md (hunk 2 above)
Comment (intls) [reply apps/hubble/www/docs/tutorials/gcp.md]: @sds, I have fixed the errors in the figure and img tags as you suggested. The pull request is now ready for review and merging. Please review the changes. Thank you!

**Pull request merged**